main_test.go

```diff
@@ -230,6 +238,83 @@ var _ = Describe("Main", func() {
 			})
 		})
 	})
+
+	Describe("metricProcessor", func() {
+		type tokenJSON struct {
```

Comment:
They're a little different. We've copied this one from the source of [`oauth2.internal`](https://github.com/golang/oauth2/blob/master/internal/token.go#L58-L64).